MagickCore/pixel-accessor.h

```diff
@@ -663,7 +663,7 @@ static inline void SetPixelb(const Image *magick_restrict image,
     pixel[image->channel_map[bPixelChannel].offset]=b;
 }
 
-static inline void SetPixelBackgoundColor(const Image *magick_restrict image,
+static inline void SetPixelBackgroundColor(const Image *magick_restrict image,
```

Comment:
This cannot be renamed because it is in a public header

Reply:
If you're up for adding a new item and deprecating, we can do that in a distinct PR.

Dropped from this one.